experimental/ner_spancat_compare/requirements.txt

```diff
@@ -0,0 +1,5 @@
+spacy==3.2.0
```

Comment:
Do we need to pin it so strictly? The `debug-data` will only have the span details from 3.3.1 onwards, so I think we could do something like >3.2 and <3.4?

Reply:
Actually, do we need to require `spacy` here at all? It should already be available when people type `spacy project run install`...

Reply:
Hehe yeah you have a point, I removed spaCy from deps